micro_ros_diagnostic_bridge/launch/diagnostic_bridge.launch.py

```diff
@@ -31,10 +31,27 @@ def generate_launch_description():
             'log_level',
             default_value=['info'],
             description='Logging level'),
+        launch.actions.DeclareLaunchArgument(
+            'remap_diagnostics_uros',
```

Comment:
Maybe ìnput_topics`? Already preparing for the fact that the bridge should listen to x topics at some point.

Reply:
I'll make the changes

Reply:
After thinking about it, I think we should stick to input_topic, and later decide what do we do about it when we have multiple. As there are multiple options, like providing two separate arguments. Or simply deprecating the single one.

What do you think?